pkg/apis/externaldns/types.go

```diff
@@ -122,7 +123,7 @@ func (cfg *Config) ParseFlags(args []string) error {
 	// Miscellaneous flags
 	app.Flag("log-format", "The format in which log messages are printed (default: text, options: text, json)").Default(defaultConfig.LogFormat).EnumVar(&cfg.LogFormat, "text", "json")
 	app.Flag("metrics-address", "Specify where to serve the metrics and health check endpoint (default: :7979)").Default(defaultConfig.MetricsAddress).StringVar(&cfg.MetricsAddress)
-	app.Flag("debug", "When enabled, increases the logging output for debugging purposes (default: disabled)").BoolVar(&cfg.Debug)
+	app.Flag("log-level", "Set the level of logging. (default: info, options: debug, info, warn, error, fatal").Default(defaultConfig.LogLevel).EnumVar(&cfg.LogLevel, logrus.DebugLevel.String(), logrus.InfoLevel.String(), logrus.WarnLevel.String(), logrus.ErrorLevel.String(), logrus.FatalLevel.String(), logrus.PanicLevel.String())
```

Comment:
I could use logrus.AllLevels instead of this... but I'd still need a function that returned a slice of strings from logrus.Levels. Happy to do it if others don't like this as much as I don't :)

Reply:
Went ahead and did this.